src/server/service/file/index.ts

```diff
@@ -26,9 +26,9 @@ export const fileTypes = {
     downloadName: `Statistical Factsheets (${levelIso})`,
     fileType: 'ods',
   }),
-  dataDownload: (key: string, fileType: string) => ({
+  dataDownload: (assessmentName: string, cycleName: string, key: string, fileType: string) => ({
     key,
-    folder: 'dataDownload',
+    folder: `${assessmentName}/${cycleName}/dataDownload`,
```

Comment:
the url was already in correct format, but the path was never used

simply moved the files to `assessmentName/cycleName/dataDownload/fileName`